src/groups/group.jl

```diff
@@ -225,25 +230,29 @@ function get_vector(M::AbstractGroupManifold, e::Identity, X, B::VeeOrthogonalBa
     return get_vector(decorated_manifold(M), e.p, X, B)
 end
 function get_vector(M::Manifold, e::Identity, X, B::VeeOrthogonalBasis)
-    M != e.group.manifold && error("On $(M) the identity $(e) does not match to perform get_vector.")
+    M != e.group.manifold &&
+    error("On $(M) the identity $(e) does not match to perform get_vector.")
```

Comment:
This is one thing the formatter does I don't like: breaking `condition && something()` lines without indenting. Maybe we should just replace them with `if` in cases where they are too long?

Reply:
Ah, yes, I think we've agreed before that way's better. Ideally in the future with the workflow, we'll all run JuliaFormatter before merging, and large runs like this where it's easy to miss these won't be necessary. If you open an issue on JuliaFormatter, they might agree and change the default.

Reply:
I created an issue for this: https://github.com/domluna/JuliaFormatter.jl/issues/222

Reply:
Thanks!